pkg/api/cluster_types.go

```diff
@@ -277,6 +278,30 @@ func (s *StrimziVersion) DeepCopy() *StrimziVersion {
 	return &res
 }
 
+// GetLatestKafkaVersion returns the latest (most recent) Kafka version
```

Comment:
@VanillaSpoon these newly added method here could be of use in the re-assignment sub-task